spec/classes/puppet_server_puppetserver_spec.rb

```diff
@@ -270,6 +272,88 @@
         end
       end
 
+      describe 'server_max_queued_requests' do
+        context 'when server_puppetserver_version >= 5.0 with default parameters' do
+          let(:params) do
+            default_params.merge(
+              :server_puppetserver_dir     => '/etc/custom/puppetserver',
+              :server_puppetserver_version => '5.0.0',
+            )
+            end
+          it 'should have max-queued-requests: 0' do
+            content = catalogue.resource('file', '/etc/custom/puppetserver/conf.d/puppetserver.conf').send(:parameters)[:content]
```

Comment:
We have a ```verify_content``` for this, or the regular ```expected.to contain_file('/etc/custom/...').with_content(/^    max-queued-requests: 0$/)``` should work.

Reply:
I've just copied it from another test in this file, seems like there are a lot of options which are used in those tests ;) Switched to `expected.to contain_file...` :)